packages/cmf/src/route.js

```diff
@@ -76,12 +76,15 @@ function loadComponents(context, item, dispatch) {
 		item.component = component.get(item.component, context);
 		if (item.view && !item.component.CMFContainer) {
 			item.component = connectView(context, item.component, item.view);
-		} else if (item.view && item.component.CMFContainer) {
+		} else if ((item.view || item.componentId) && item.component.CMFContainer) {
 			const WithView = item.component;
-			item.component = props => <WithView view={item.view} {...props} />;
+			item.component = props => (
```

Comment:
really nice, could you please add a console.warn if item.view ?
`console.warn('DEPRECATION: please replace view by componentId and update your settings')`